extra-tools/index.md

```diff
@@ -78,9 +83,9 @@ Note that this filter rewrites the `group` key from each item. For instance, an
 
 ## Shortcut icon (`shortcutIcon`)
 
-When used, the `shortcutIcon` filter takes the eponymous key from the configuration and converts it into an object with `type`, `url` and `path` keys.
+The `shortcutIcon` filter takes the eponymous key from the configuration and converts it into an object with `type`, `url` and `path` keys.
 
-For instance, `"http://absolute.path/to/icon.png"` as a value in `ctx.shortcutIcon` would yield `ctx.shortcutIcon` as:
+To use the icon URL `http://absolute.path/to/icon.png` in `ctx.shortcutIcon`:
```

Comment:
The code example underneath is the result of the transformation after the filter is applied, and not an usage example.
So this is wrong.